Makefile

```diff
@@ -349,13 +349,26 @@ devdoc-update:
 ###                                Protobuf                                 ###
 ###############################################################################
 
+# Get clang-format binary name
+ifeq ($(UNAME_S),Linux)
+# Checking debian release or not
+ifneq ("$(wildcard /etc/debian_version)","")
```

Comment:
Please don't do this. This works on Debian and derivates only. It does not work on other distros.